### PR TITLE
[CORE-5998] Add note about obsolete amenities

### PR DESCRIPTION
**Pull request by pyromaniac**
Demo: https://a.cl.ly/geuE0kY8

### CODE REVIEW

**Review by ston1x — APPROVED** [review]
Looks good to me!
Just wondering: why did you remove [content/reference/changelog.md](https://github.com/BookingSync/bookingsync-api-docs/pull/380/files#diff-a6ff0625a1d123b34bff0f92017f677d426129ddeb9f918f53dce8af2adabe49)?

File: content/reference/endpoints/rentals_amenities.md [hunk 1]
```diff
@@ -7,6 +7,16 @@
...
 This resource represents the Rentals Amenities, that establish relationship between [Rentals](/reference/endpoints/rentals) and [Amenities](/reference/endpoints/amenities).
 
+## Obsolete amenities
+
+<div class="callout callout-warning" markdown="1">
+  Right now, 3 amenities are marked as obsolete and are replaced with [Rental#checkin_type](/reference/endpoints/rentals#checkin-type) attribute:
+`contactless_check_in_and_check_out`, `self_checkin`, and `24h_checkin`. It is impossible to assign those amenities to
+rentals directly or unassign them from rentals.
+
+  Instead, these 3 amenities are assigned/unassigned automatically on [Rental#checkin_type](/reference/endpoints/rentals#checkin-type) update.
```
Comment (adamgrad) [inline content/reference/endpoints/rentals_amenities.md:17]: Good to mention that this is going to happen until full deprecation and removal of those amenities.

**Review by adamgrad — COMMENTED** [review]
@pyromaniac Looks good! Same question as from Nicolai, what happened with changelog.md?

**Comment by pyromaniac** [comment]
changelog.md is an old concept, we are not linking it anywhere, the last record is old. Instead, we are linking github releases page now.

**Review by adamgrad — APPROVED** [review]

**Pull request merged**